test/blackbox-tests/test-cases/variants/run.t

```diff
@@ -147,7 +147,7 @@ Virtual library where a wrapped module is virtual
 Executable that tries to build against a virtual library without an implementation
   $ dune build --root missing-implementation
   Entering directory 'missing-implementation'
-  Error: No implementation found for virtual library "vlib" (_build/default/vlib).
+  Error: No implementation found for virtual library "vlib" (_build/default/vlib).-> required by executable foo in dune:1
```

Comment:
I think a newline is missing here.

Reply:
Indeed, the formatting was off. I fixed it.